solidity/contracts/BurntPixArchives.sol

```diff
@@ -99,13 +99,19 @@ contract BurntPixArchives is LSP8CappedSupply {
     }
 
     function refineToArchive(uint256 iters, address contributor) public {
-        require(iters > 0);
+        //should we stop refining if the original is claimed?
+        require(iters > 0); //do we want a minimum iteration amount?
```

Comment:
i think anything greater than 0 is good, mainly cause these things could be quite costly and at times you might literally need just a single iteration to unlock the next archive level